tools/tests/package.json

```diff
@@ -0,0 +1,26 @@
+{

```

Comment:
Can you add the `coherent-gameface-accordion-menu` component here before you merge when rebase with master?

Reply:
Yes.